org.eclipse.xtext.maven.plugin/src/main/java/org/eclipse/xtext/maven/AbstractXtextGeneratorMojo.java

```diff
@@ -87,10 +90,10 @@ public String getEncoding() {
 	private List<ProjectMapping> projectMappings;
 
 	@Parameter(property = "xtext.generator.skip", defaultValue = "false")
-	private Boolean skip;
+	private boolean skip;
 
 	@Parameter(defaultValue = "true")
-	private Boolean failOnValidationError;
+	private boolean failOnValidationError;
```

Comment:
I think this should be `= true` (similar to another one below), just in case.

Reply:
I changed it too.